### ISSUE TITLE
Link in-game help to community wiki

### ISSUE THREAD

**Issue by ACG8** [sIKE23/Mage-Wars#211]
See this thread: http://forum.arcanewonders.com/index.php?topic=15365.0

I figured out how to read content off of a webpage. This should make gathering content much easier.

**Comment by sIKE23** [comment]
Do you want to get him the txt files you have written at this point?

**Comment by ACG8** [comment]
Good point. Yes, I think I should do that.

Once the format is determined, incorporating this should be very easy, so I am thinking it could be done as a patch to the Q1 release (which is tomorrow!); we don't want the wiki to stagnate.

**Comment by sIKE23** [comment]
yeah and it looks like there will be a new OCTGN build here soon and I was
told the card back code could make it in this release.....

**Comment by sIKE23** [comment]
Created a folder called For-Wiki and put you two docs in along with a
spreadsheet matching card names to URLs.

Let me know if it is ok to share this with him.

**Comment by ACG8** [comment]
Yes, please do. That should give him a big start on the wiki.
